code/datums/greyscale/config_types/greyscale_configs/greyscale_clothes.dm

```diff
@@ -363,6 +363,16 @@ SKYRAT EDIT END */
 	icon_file = 'icons/mob/clothing/under/shorts_pants_shirts.dmi'
 	json_config = 'code/datums/greyscale/json_configs/buttondown_skirt_worn.json'
 
+/datum/greyscale_config/cso //Bubber EDIT - Command stripper outfits.
+	name = "command stripper outfit"
+	icon_file = 'modular_zubbers/icons/obj/clothing/under/captain.dmi'
+	json_config = 'code/datums/greyscale/json_configs/cso.json'
+
+/datum/greyscale_config/cso/worn
+	name = "command stripper outfit (worn)"
+	icon_file = 'modular_zubbers/icons/mob/clothing/under/captain.dmi'
+	json_config = 'code/datums/greyscale/json_configs/cso.json'
+
```

Comment:
This can be in a modular folder

Reply:
Do you have any current modularity for GAGS?